File: DayFour/Java/TwoSumTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertArrayEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;

public class TwoSumTest{

    @Test
    
    public void testIPassAnArrayAndANumberAndItGivesMeNumbersThatSumToGivenNumber(){
    TwoSum calc = new TwoSum();
    int[] array = {2,5,6,7,-6};
    int number = 8;
    int[] actual = TwoSum.getNumbers(array, number);
    int[] expected = {2, 6};
    assertArrayEquals(actual, expected);
    }
    
    @Test
    
    public void testIPassAnArrayAndANumberAndItGivesMeNumbersThatSumToGivenNumberAgain(){
    TwoSum calc = new TwoSum();
    int[] array = {2,5,6,3,-6};
    int number = 8;
    int[] actual = TwoSum.getNumbers(array, number);
    int[] expected = {2, 6};
    assertArrayEquals(actual, expected);
    }
}

File: DayFour/Python/two_sum_test.py
from unittest import TestCase
from two_sum import *

class TwoSumTest(TestCase):
    def test_that_i_pass_an_array_and_number_i_get_numbers_to_that_give_sum(self):
        array = [2, 5, 6, 7, -6]
        number = 8
        actual = get_numbers(array, number)
        expected = [2, 6]
        self.assertEqual(actual, expected)









